app/src/main/java/org/kiwix/kiwixmobile/di/modules/ServiceModule.kt

```diff
@@ -38,10 +39,9 @@ class ServiceModule {
   @Provides
   @ServiceScope
   fun providesWebServerHelper(
-    jniKiwixLibrary: Library,
-    kiwixServer: JNIKiwixServer,
+    kiwixServer: KiwixServer,
```

Comment:
this will want to be the factory, KiwixServer will either want to be a `lateinit val` or a nullable var

Reply:
ugh, right Java. Well it'll just be a private field

Reply:
Yep!